FindMagicNumber/Program.cs

```diff
@@ -8,11 +8,143 @@ namespace FindMagicNumber
+                                magicNumsx3 = IntToArray(i * 3);
+                                if (CompareArrays(magicNums, magicNumsx3) == true)
+                                {
+                                    if (ArrayCheck(IntToArray(i * 4)) == true && i * 4 < 999999)
+                                    {
+                                        num4 = i * 4;
+                                        magicNumsx4 = IntToArray(i * 4);
+                                        if (CompareArrays(magicNums, magicNumsx4) == true)
+                                        {
+                                            if (ArrayCheck(IntToArray(i * 5)) == true)
+                                            {
+                                                num5 = i * 5;
+                                                magicNumsx5 = IntToArray(i * 5);
+                                                if (CompareArrays(magicNums, magicNumsx5) == true)
+                                                {
+                                                    if (ArrayCheck(IntToArray(i * 6)) == true)
+                                                    {
+                                                        num6 = i * 6;
+                                                        magicNumsx6 = IntToArray(i * 6);
+                                                        if (CompareArrays(magicNums, magicNumsx6) == true)
+                                                        {
+                                                            Console.WriteLine(num1);
+                                                            Console.WriteLine(num2);
+                                                            Console.WriteLine(num3);
+                                                            Console.WriteLine(num4);
+                                                            Console.WriteLine(num5);
+                                                            Console.WriteLine(num6);
+
+                                                        }
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+
+
+        }
+        static int[] IntToArray(int num)
         {
-            Console.WriteLine("Hello World!");
+            int[] magicNums = new int[6];
+            magicNums[0] = num / 100000;
+            magicNums[1] = num / 10000 % 10;
+            magicNums[2] = num / 1000 % 10;
+            magicNums[3] = num / 100 % 10;
+            magicNums[4] = num / 10 % 10;
+            magicNums[5] = num % 10;
+            return magicNums;
+        }
+
+        static bool ArrayCheck(int[] array)
```

Comment:
Pilnai pataisyta funkcija turetu atrodyti taip: 

        static bool ArrayCheck(int[] array)
        {
            bool rez = true;
            for (int i = 0; i < array.Length && rez; i++)
            {
                for (int j = i + 1; j < array.Length; j++)
                {
                    if (array[i] == array[j])
                    {
                        rez = false;
                        break;
                    }
                }
            }
            return rez;
        }